ci/generate_json.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import os.path
+import sys
+
+repo_root = os.path.join(os.path.dirname(__file__), "..")
+sys.path.append(os.path.join(repo_root, "src"))
```

Comment:
I *think* you  might be able to avoid this path-merging by just adding the package's source code to `PYTHONPATH` from the pre-commit configuration, like this:

```yaml
entry: PYTHONPATH=src/ ./ci/generate_json.py
```

(assuming `entry` is just passed through and exec'd, which I don't know for sure)

If that's possible, I think it'd be a little simpler.

But this is a minor suggestion, do whatever you want with it.

Reply:
Doesn't work. When I try that I get:

```
Executable `PYTHONPATH=src/` not found
```

Reply:
alright it was worth a shot, thanks for trying